### DISCUSSION TITLE
Custom Pricing App is available now?

### DISCUSSION THREAD

**Discussion by scrdongmin** [saleor/saleor#17116] [Q&A]
Hi, I'm currently testing to use the cloud version of Saleor.

We are a company where quantity-based pricing is important, so I checked the [cookbook](https://docs.saleor.io/developer/products/cookbook) and found the Github like below.
I followed the instructions and went through the installation process normally.
![Screen_11-30(i6KoYoen)](https://github.com/user-attachments/assets/fadc21ce-2879-4084-bc00-56fe1f198c15)

However, the installation result keeps saying Fail. Can I assume that the app is currently unavailable?

[https://github.com/krzysztofwolski/saleor-app-checkout-prices](https://github.com/krzysztofwolski/saleor-app-checkout-prices)

**Comment by tomaszlud** [comment]
Hi, could you share the logs from the application you're trying to install? The 403 error seems to be coming from the `saleor-app-checkout-prices` local instance.

**Reply by danus12** [reply]
Hi guys
Maybe my question is not matching the subject
But I want to add additionnal fees for credit card fees, how can I do it ? I mean easiest way
